### ISSUE TITLE
The problem of structure quantity generated and loss function.

### ISSUE THREAD

**Issue by JHWang1001** [txie-93/cdvae#66]
Dear txie,

I train the `pervo` data use 
```
python cdvae/run.py data=perov expname=perov model.predict_property=True
python scripts/evaluate.py --model_path MODEL_PATH --tasks  opt
```
I have some question:
```
train_loss_step 7.70267
train_kld_loss_step 32.16307
train_kld_loss_epoch 33.9217
```
These numbers look like I didn't get a good convergence results. 
Does this mean that my training wasn't successful? 
Can you give me some examples for reference? 
Are they related to the number of structures generated? 
I only got 356 structures after train `pervo`. Sometime, I use my own data, I only can get 11 structures. XD. Can I get more structures with more training step?@txie-93 

Best wishes and looking forward to your reply
Jiahui Wang
